signing_clients/tests/__init__.py

```diff
@@ -97,9 +102,17 @@ def test_file(fname):
 
 class SigningTest(unittest.TestCase):
 
+    def setUp(self):
+        self.tmpdir = tempfile.mkdtemp()
+
+    def cleanUp(self):
```

Comment:
I don't think this hook is called by unittest. I don't see it in the docs anyway.